website/docusaurus.config.js

```diff
@@ -66,7 +71,11 @@ const config = {
             position: 'left',
             label: 'Documentation',
           },
-          // {to: '/blog', label: 'Blog', position: 'left'},
+          // {
```

Comment:
@AlekSi  Here, I need to remove the comment around the blog but can't do that without getting errors. We need the default content plugin [here](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-blog) to use the blog.

Reply:
Please send PR to https://github.com/FerretDB/deps to add that dependency

Reply:
More specifically, that file should be updated: https://github.com/FerretDB/deps/blob/main/docusaurus-docs/package.json

Reply:
Looking through again, I think this dependency already exists, it comes with this one `@docusaurus/preset-classic`. At the moment, I think there's no way to route to the path: `website/blog`.  Is it possible the problem is from there? 

It worked when I added this `- ./website/blog:/workdir/docusaurus-docs/blog:ro` to the Docker compose file [here](https://github.com/FerretDB/FerretDB/blob/main/docker-compose.yml?plain=1#L129-L135).

Reply:
Docker Compose configuration change looks correct, please add it to this PR

Reply:
PTAL